fortepyan/midi/structures.py

```diff
@@ -129,6 +129,9 @@ def trim(self, start: float, finish: float, shift_time: bool = True, slice_type:
             Trimming with time shift disabled:
             >>> midi_piece.trim(start=1.0, finish=5.0, shift_time=False)
 
+            An example of a trimmed MIDI piece:
+            ![Trimmed MIDI piece](../assets/random_midi_piece.png)
```

Comment:
Shortly speaking, it's possible to add an image to the documentation. relative links work just fine, it is, however, a little tricky to guess what will be the final path.